hexrd/ui/indexing/fit_grains_results_dialog.py

```diff
@@ -168,6 +169,25 @@ def on_export_button_pressed(self):
 
             self.data_model.save(selected_file)
 
+    def on_export_stresses_button_pressed(self):
+        if self.tensor_type != 'stress':
+            raise Exception('Tensor type must be stress')
+
+        selected_file, selected_filter = QFileDialog.getSaveFileName(
+            self.ui, 'Export Fit-Grains Stresses', HexrdConfig().working_dir,
+            'Npz Files (*.npz)')
+
+        if not selected_file:
+            return
+
+        stresses = self.converted_data[:, 15:21]
```

Comment:
It would be useful to add a comment or define constants for 15 and 21, to make it clear what we a slicing here.

Reply:
Sounds good, I'll do that soon.